src/_designs/components/appDefinition/index.vue

```diff
@@ -57,7 +57,8 @@ export default {
   }
   .app-definition-content {
     color: #76818C;
-    @extend %face-tracked-base;
+    display: flex;
+    @extend %face-mono-xs;
```

Comment:
I think as a general rule everywhere, putting any `@extend` or `@import` in the header of the `.css-class` would be the best. Since you can overwrite them easily, as css cascases.

Reply:
sounds good